src/components/Standings/mapTournamentToStandings.ts

```diff
@@ -24,19 +25,26 @@ export default function mapTournamentToStandings(
           {
             teamId: team.id,
             teamName: team.name,
-            score: 0
-          }
+            score: 0,
+            gamesPlayed: 0,
+          },
         ];
       }
     });
 
     const maybeWinnerTeamId = match.result;
-
     return newStandingsSoFar.map(standing => {
-      if (standing.teamId !== maybeWinnerTeamId) {
-        return standing;
-      }
-      return { ...standing, score: standing.score + 1 };
+      const deltaGamesPlayed =
+        match.teams.map(team => team.id).includes(standing.teamId) &&
+        maybeWinnerTeamId !== "not played"
+          ? 1
+          : 0;
+      const deltaScore = standing.teamId === maybeWinnerTeamId ? 1 : 0;
+      return {
+        ...standing,
+        score: standing.score + deltaScore,
+        gamesPlayed: standing.gamesPlayed + deltaGamesPlayed,
+      };
```

Comment:
It's not even part of this PR, but just to brag about functional programming I'll say it. You can avoid using the `let ` for the `newStandingsSoFar` variable with something like the following:

```
const standingsWithNoGames: Standing[] = match.teams
  .filter(
    team => !standingsSoFar.some(standing => standing.teamId === team.id)
  )
  .map(team => ({
    teamId: team.id,
    teamName: team.name,
    score: 0,
    gamesPlayed: 0
  }));

const newStandingsSoFar: Standing[] = [
  ...standingsSoFar,
  ...standingsWithNoGames
];
```

Reply:
Changed